command/ssh/certificate.go

```diff
@@ -254,6 +261,28 @@ func certificateAction(ctx *cli.Context) error {
 		if err != nil {
 			return err
 		}
+
+		// All host identity certs need a URI SAN to work with our ssh API.
+		if isHost {
+			var u = uuid.Nil
+			if hostID == "" {
+				u, err = deriveMachineID()
+				if err != nil {
+					return err
```

Comment:
We should return an error saying something like `cannot read machine-id, please use the --host-id flag`

Reply:
good idea

Reply:
done

Reply:
done